curvefs/src/client/s3/client_s3_adaptor.cpp

```diff
@@ -369,5 +377,25 @@ int S3ClientAdaptorImpl::ClearDiskCache(int64_t inodeId) {
     return ret;
 }
 
+void S3ClientAdaptorImpl::Enqueue(
+  std::shared_ptr<FlushChunkCacheContext> context) {
+    auto task = [this, context]() {
+        this->FlushChunkClosure(context);
+    };
+    taskPool_.Enqueue(task);
+}
+
+int S3ClientAdaptorImpl::FlushChunkClosure(
+  std::shared_ptr<FlushChunkCacheContext> context) {
+    VLOG(9) << "FlushChunkCacheClosure start: " << context->inode;
+    FlushChunkCacheClosure done(context);
```

Comment:
If FlushChunkCacheClosure is only used here, so there is no need to package such a class, just call ctx->cb(ctx)  directly

Reply:
fix